x-pack/plugins/security_solution/public/resolver/view/use_state_syncing_actions.ts

```diff
@@ -29,20 +29,20 @@ export function useStateSyncingActions({
   shouldUpdate: boolean;
   filters: object;
 }) {
-  const dispatch = useResolverDispatch();
+  const dispatch = useDispatch();
   const locationSearch = useLocation().search;
   useLayoutEffect(() => {
-    dispatch({
-      type: 'appReceivedNewExternalProperties',
-      payload: {
+    dispatch(
+      appReceivedNewExternalProperties({
```

Comment:
@kqualters-elastic might know more, but I think these actions are used to bring data modeled in the security solution store into the resolver store. Is this still needed?